test/helpers.ts

```diff
@@ -174,3 +198,5 @@ export const cleanupTaxonomy = async (olderThan = 1000 * 60 * 60) => {
     )
   )
 }
+
+export const timeoutToCalmRateLimiting = () => new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
ha, nice 😎

Reply:
It did calculate at some point, wanted to catch any rate-limits and set the value to the given delay till we can call the API again.

But that never triggered, so I kept the default 1000ms... maybe rename the function now?